websites/D/Daum/presence.ts

```diff
@@ -0,0 +1,394 @@
+
+            presenceData.details = "Watching video:";
+            presenceData.state = video.title ?? document.querySelector("h3.tit_view")?.textContent;
+
+            presenceData.smallImageKey = video.paused ? "pause" : "play";
+            presenceData.smallImageText = video.paused ? "Paused" : "Playing";
+
+            presenceData.startTimestamp = timestamps[0];
+            presenceData.endTimestamp = timestamps[1];
+
+            presenceData.buttons = [{
+              label: "Watch Video",
+              url: document.baseURI
+            }];
+
+            if (video.paused){
+              delete presenceData.startTimestamp;
+              delete presenceData.endTimestamp;
+            }
+          }
+        }
+      },
+      "/([a-zA-Z0-9]+)/([a-zA-Z0-9]+)":{
+        service: "DAUM_BLOG",
+        setPresenceData(){
+          presenceData.details = "Reading blog:";
+          presenceData.state = document.querySelector("a.link-title")?.textContent;
+
+          presenceData.buttons = [{
+            label: "Read Blog",
+            url: document.baseURI
+          }];
+        }
+      },
+      "vod/view/([0-9a-zA-Z]+)": {
+        service: "DAUM_AUTO",
+        setPresenceData(){
+          const timestamps = presence.getTimestamps(video?.currentTime, video?.duration);
+          
+          presenceData.details = "Watching video:";
+          presenceData.state = document.querySelector("strong.tit_vod")?.textContent;
+
+          presenceData.smallImageKey = video?.paused ? "pause" : "play";
+          presenceData.smallImageText = video?.paused ? "Paused" : "Playing";
+
+          presenceData.startTimestamp = timestamps[0];
+          presenceData.endTimestamp = timestamps[1];
+
+          presenceData.buttons = [{
+            label: "Watch Video",
+            url: document.baseURI
+          }];
+
+          if (video?.paused){
+            delete presenceData.startTimestamp;
+            delete presenceData.endTimestamp;
+          }
+        }
+      },
+      "/newcar/model/([0-9a-zA-Z]+)": {
+        service: "DAUM_AUTO",
+        setPresenceData(){
+          presenceData.details = "Viewing car:";
+          presenceData.state = document.querySelector("h3.tit_model")?.textContent;
+
+          presenceData.buttons = [{
+            label: "View Car",
+            url: document.baseURI
+          }];
+        }
+      },
+      "/moviedb/": {
+        service: "DAUM_MOVIE",
+        setPresenceData(){
+          presenceData.details = "Viewing movie:";
+          presenceData.state = document.querySelector('span.txt_tit')?.textContent;
+
+          presenceData.buttons = [{
+            label: "View Movie",
+            url: document.baseURI
+          }];
+        }
+      },
+      "/person/": {
+        service: "DAUM_MOVIE",
+        setPresenceData(){
+          presenceData.details = "Viewing cast:";
+          presenceData.state = document.querySelector('span.txt_tit')?.textContent;
+
```

Comment:
shouldnt it be
"Viewing cast of: {show}"

Reply:
It's actually one person, I'm probably the one who mislead here LOL